src/xen_helper/xen_helper.c

```diff
@@ -172,6 +178,10 @@ void xen_free_interface(xen_interface_t* xen)
             xc_interface_close(xen->xc);
         if (xen->evtchn)
             xc_evtchn_close(xen->evtchn);
+#ifdef ENABLE_IPT
```

Comment:
No need for ifdef here, `xen-fmem` will be null if it's not enabled.

Reply:
fixed